criu/sk-unix.c

```diff
@@ -2225,8 +2225,8 @@ int add_fake_unix_queuers(void)
 	list_for_each_entry(ui, &unix_sockets, list) {
 		if ((ui->ue->uflags & (USK_EXTERN | USK_CALLBACK)) || ui->queuer)
 			continue;
```

Comment:
here we check that queuer isn't nil, why do we need to check it again below?

Reply:
note: queuer != peer